modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring namespace-scoped Argo Rollouts installation
+
+The following example provides the steps to configure namespace-scoped Argo Rollouts installation.
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed {gitops-title} on your {OCP} cluster.
+
+.Procedure 
+
+. In the *Administrator* perspective of the web console, go to *Administration* -> *CustomResourceDefinitions*.
+. Search for `Subscription` and click the `Subscription` CRD.
+. Click the *Instances* tab and then click the `openshift-gitops-operator` subscription.
+. Click the *YAML* tab and edit the YAML file. 
+. Specify the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` environment variable, with the value set to `true` in the `.spec.config.env` property.
++
+.Example of configuring the namespace-scoped Argo Rollouts installation
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+spec:
+  # (...)
+  config:
+    env:
+      - name: NAMESPACE_SCOPED_ARGO_ROLLOUTS
+        value: 'true' #<1>
+----
+<1> Set value to `true` to enable namespace-scoped installation. If the value is set to `false` or not specified, the installation defaults to cluster-scoped mode.
+
+. Click *Save*.
++
+As a result, the {gitops-title} Operator enables the namespace-scoped Argo Rollouts installation to reconcile Argo Rollouts CR.
+
+. Verify if the {gitops-title} Operator has enabled namespace-scoped Argo Rollouts installation by viewing the logs of the {gitops-shortname} container:
+.. In the *Administrator* perspective of the web console, go to *Workloads* -> *Pods*.
+.. Click the `openshift-gitops-operator-controller-manager` pod, and then click the *Logs* tab.
+.. Look for the following log statement: `Running in namespaced-scoped mode`. This indicates that the {gitops-title} Operator has enabled the namespace-scoped Argo Rollouts installation.
+
+. Create a `RolloutManager` resource to perform the namespace-scoped Argo Rollout installation:
+.. Create or select the project where you want to perform the namespace-scoped Argo Rollout installation from the *Project* drop-down menu.
+.. Go to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps*, and click the *RolloutManager* tab.
+.. Click the *Create RolloutManager* button.
+.. Select *YAML view* and enter the following snippet:
++
+.Example RolloutManager CR for namespace-scoped Argo Rollouts installation
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: rollout-manager
+spec:
+  namespaceScoped: true
+----
```

Comment:
```suggestion
----
<1> Specify the namespace where you want to perform the namespace-scoped Argo Rollout installation.
```

Reply:
@Dhruv-Soni11 One more minor suggestion here regarding this: https://github.com/openshift/openshift-docs/pull/78933#discussion_r1692697916

Reply:
to make the namespace generic

Reply:
Fixed.